static/css/style.css

```diff
@@ -1,15 +1,16 @@
 /* Fonts. */
 /* Colors. */
 /* Background colors. */
-/*Hover color */
+/* Hover color. */
```

Comment:
Would be better if this is not shown in the css, but only in the sass file.
Please try replacing the comments from C-style block (/* */) to an inline comment style (//) **in the SASS file**. The inline comment style will not appear in the CSS file, only in SASS.

Reply:
Done